Change quickstarts to use new secrets format (#137)
spiceai/quickstarts: rds-aurora-mysql/README.md

**Step 3.** Depending on chosen credentials management option (AWS Secrets Manager, or Self managed) retrieve `username` and `password`. 

> **Step 4.** Edit the `spicepod.yaml` file in this directory and replace `[remote_table_path]` with the path to the remote table to be accelerated, `[local_table_name]` with your desired name for the locally accelerated table, and the `[mysql_host]` and `[mysql_tcp_port]` params with the connection parameters from your AWS RDS instance. The `[mysql_user]` should be set to the username for the RDS instance. The `[mysql_db]` should be set to the name of the database in the RDS instance. The `MYSQL_PASS` environment variable should be set to the password for the RDS instance. This can be specified on the command line when running the Spice runtime, or in a `.env` file in the same directory as `spicepod.yaml`.

Reviewer: ```suggestion
**Step 4.** Edit the `spicepod.yaml` file in this working directory and replace `[remote_table_path]` with the path to the remote table to be accelerated, `[local_table_name]` with the desired name for the locally accelerated table, and the `[mysql_host]` and `[mysql_tcp_port]` params with the connection parameters for the AWS RDS instance. The `[mysql_user]` should be set to the username for the RDS instance. The `[mysql_db]` should be set to the name of the database in the RDS instance. The `MYSQL_PASS` environment variable should be set to the password for the RDS instance. Environment variables can be specified on the command line when running the Spice runtime or in a `.env` file in the same directory as `spicepod.yaml`.
```
